modules/disabling-insights-operator-alerts.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-operator.adoc
+
+
+:_content-type: CONCEPT
+[id="disabling-insights-operator-alerts_{context}"]
+= Disabling Insights Operator alerts
+
+You can stop Insights Operator from firing alerts to the cluster prometheus instance.
+
+. Navigate to *Workloads* -> *Secrets*.
+. Click *Show default projects*.
```

Comment:
After step 1, a new user may not immediately know where to find the *Show default projects* . Consider updating the procedure with the following step. Also, see guidance from IBM Style on [switch](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#switch)
```suggestion
. On the *Secrets* page, select *All Projects* from the *Project* list, and then set *Show default projects* to on.
```